Frontend/Skyblock-Global-Chat-CT/src/index.js

```diff
@@ -1,7 +1,76 @@
 import { connection } from "./serverConnection";
+import { ChatPrefix, RED, YELLOW, GOLD, RESET, GREEN } from "./utils/constants";
+import settings from "./settings";
+
+let globalChatEnabled = false;
+
+register("command", (...args) => {
+  if (args.length < 1) {
+    ChatLib.chat(`${ChatPrefix} ${RED}Usage: /globalchat <message>${RESET}`);
+    return;
+  }
+  const message = args.join(" ");
+
+  const messageData = {
+    User: Player.getName(),
+    Text: message
+  };
+  connection.send(JSON.stringify(messageData));
+}).setName("globalchat");
+
+register("command", (...args) => {
+  if (!args) return;
+  if (!args[0]) {
+    settings.openGUI();
+    return;
+  }
+
+  const subCommand = (args[0] || "").toLowerCase();
+  switch (subCommand) {
+    case "settings":
+      settings.openGUI();
+      break;
+    case "connect":
+      connection.connect();
+      break;
+    case "disconnect":
+      connection.close();
+      ChatLib.chat(`${ChatPrefix} ${RED}Disconnected. Auto-reconnect disabled.${RESET}`);
+      break;
+    case "toggle":
+      globalChatEnabled = !globalChatEnabled;
+      const globalChatEnabledStatus = globalChatEnabled ? `${GREEN}Enabled${RESET}` : `${RED}Disabled${RESET}`;
```

Comment:
change to 
` globalChatEnabled = !globalChatEnabled;

      const globalChatEnabledStatus = globalChatEnabled ? `${GREEN}Enabled${RESET}` : `${RED}Disabled${RESET}`;
Comment
      ChatLib.chat(`${ChatPrefix} ${GOLD}Auto-send to global is now:${RESET} ${globalChatEnabledStatus}`);
      break;`
to make it more visible that globalChatEnabledStatus is used for the chat send (whitespace is useful if used correctly, and can confuse if used incorrectly)